html/modules/custom/reliefweb_import/src/Plugin/ReliefWebImporter/WfpLogClusterImporter.php

```diff
@@ -0,0 +1,583 @@
+          '@id' => $id,
+          '@entity_id' => $entity->id(),
+        ]));
+      }
+      catch (\Exception $exception) {
+        $import_record['status'] = 'error';
+        $import_record['message'] = $exception->getMessage();
+        $import_record['attempts'] = ($import_record['attempts'] ?? 0) + 1;
+        $this->getLogger()->error(strtr('Unable to process WFP Logcluster document @id: @exception', [
+          '@id' => $id,
+          '@exception' => $exception->getMessage(),
+        ]));
+      }
+      finally {
+        $import_records[$import_record['imported_item_uuid']] = $import_record;
+      }
+    }
+
+    // Create or update the import records.
+    $this->saveImportRecords($import_records);
+
+    return $processed;
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  protected function processDocumentData(string $uuid, array $document): array {
+    $data = [];
+
+    // Source: Logistics Cluster.
+    $sources = [3594];
+
+    if (isset($document['organisations'])) {
+      foreach ($document['organisations'] ?? [] as $organisation) {
+        if ($organisation === 'WFP') {
+          continue;
+        }
+
+        if ($source = $this->getSourceByName($organisation)) {
+          $sources[] = $source;
+        }
+      }
+    }
+
+    // Document URL.
+    $url = $document['path'];
+
+    // Retrieve the title and clean it.
+    $title = $this->sanitizeText($document['title'] ?? '');
+
+    // No body text is available in the API for the WFP Logcluster documents.
+    $body = '';
+
+    // Retrieve the publication date.
+    $published = strtotime($document['last_update'] ?? $document['date']);
+    $published = DateHelper::format($published, 'custom', 'c');
+
+    // Retrieve the document languages and default to English if none of the
+    // supported languages were found.
+    $languages = [];
+    if (isset($document['document_language'])) {
+      if (isset($this->languageMapping[$document['document_language']])) {
+        $languages[$document['document_language']] = $this->languageMapping[$document['document_language']];
+      }
+    }
+
+    if (empty($languages)) {
+      $languages['English'] = $this->languageMapping['English'];
+    }
+
+    // Retrieve the content format and map it to 'Other' if there is no match.
+    foreach ($document['document_type'] ?? [] as $type) {
+      if (isset($this->formatMapping[$type])) {
+        $formats = [$this->formatMapping[$type]];
+        break;
+      }
+    }
+
+    if (empty($formats)) {
+      $formats = [9];
+    }
+
+    // Retrieve the countries. Consider the first one as the primary country.
+    $countries = [];
+    foreach ($document['countries'] ?? [] as $location) {
+      if (isset($this->countryMapping[$location])) {
+        $country = $this->getCountryByIso($location);
+        $countries[] = $country;
+        $countries = array_unique($countries);
+      }
```

Comment:
```suggestion
      $country = $this->getCountryByIso($location);
      $countries[] = $country;
      $countries = array_unique($countries);
```

There is no `countryMapping` and the location is an ISO3 code.